Building 2.5 fails on (emulated) arm64/aarch64

The build fails with:
```
…
   Compiling yara v0.30.0
error[E0609]: no field `xor_key` on type `&YR_MATCH`
  --> /usr/local/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/yara-0.30.0/src/internals/matches.rs:54:24
   |
54 |             xor_key: m.xor_key,
   |                        ^^^^^^^ unknown field
   |
   = note: available fields are: `base`, `offset`, `match_length`, `data_length`, `data` ... and 4 others
```


As a side note, would you mind releasing arm64 binary files ?

--- a/documentation/user_documentation/administration.rst
+++ b/documentation/user_documentation/administration.rst
@@ -238,9 +238,10 @@ This parameter tells if **Keysas** should remove the file if Yara matched at lea
 ALLOWED_TYPES
 ~~~~~~~~~~~~~~~
 
-This parameter creates a whitelist of allowed file types. Types not explicitly listed here simply won't by transfered. For now the following types are supported:
-
-https://github.com/bojand/infer#supported-types
+This parameter creates a whitelist of allowed file types. Types not explicitly
+listed here simply won't by transfered. The type are infered from the content
+of the file (not from their extension). The known types and associated mime
+type are listed here: https://github.com/bojand/infer#supported-types
 
 keysas-out
 --------------

--- a/documentation/user_documentation/installation.rst
+++ b/documentation/user_documentation/installation.rst
@@ -37,25 +37,21 @@ To compile Keysas from sources, let's start by installing the required dependenc
 Getting **Keysas**
 -------------------
 
-A pre-compiled **Keysas** binary is at your
+A pre-compiled **Keysas** binary for ``x86_64`` architecture is at your
 disposal. We recommend using the latest version here:
 https://github.com/r3dlight/keysas/releases
 
 
 Download the following files of lastest stable version.
- * keysas-vx.y.z.zip
- * keysas-vx.y.z.zip.sha256
- * keysas-vx.y.z.zip.sig
+ * keysas-vx.y.z.-x86_64.zip
+ * keysas-vx.y.z.-x86_64.zip.sha256
 
-First, verify the sha256sum and compare it to the keysas-vx.y.z.zip.sha256
-file, and import our public gpg key:
+Verify the sha256sum and compare it to the keysas-vx.y.z.zip.sha256
+file:
 
 .. code-block:: shell-session
 
  $ diff <(sha256sum keysas-vx.y.z.zip) keysas-vx.y.z.zip.sha256 
- $ wget https://keysas.fr/download/public.gpg
- $ gpg2 --import public.gpg
- $ gpg2 --verify keysas-vx.y.z.zip.sig keysas-vx.y.z.zip
  $ unzip keysas-vx.y.z.zip -d keysas
 
 

--- a/documentation/user_documentation/installation.rst
+++ b/documentation/user_documentation/installation.rst
@@ -145,6 +145,15 @@ To install the Full USB version of **Keysas** (decontamination station):
  $ sudo make install
  $ sudo make install-yararules
 
+.. admonition:: False positive detection by Yara rules
+  :class: note
+
+  Some Yara rules (for example the **Big_numbers*** series) may give repeated false
+  positives on some document types. If you want to remove them, you'll have to
+  locate them in **/usr/share/keysas/rules**, commenting them out and running
+  the **index_gen.sh** script.
+
+
 At the end of the installation, you should see something like this:
 
 .. image:: /img/install_completed.png 

--- a/documentation/user_documentation/installation.rst
+++ b/documentation/user_documentation/installation.rst
@@ -182,6 +182,52 @@ If you want to check the full installation (USB mode):
 
  $ systemctl status keysas keysas-in keysas-transit keysas-out keysas-io keysas-backend
 
+Cross compiling for RPi4
+------------------------
+
+If you don't have a RPi4 with all the needed dependencies at hand, you may
+try to cross compile via docker and qemu via `multiarch/qemu-user-static
+<https://github.com/multiarch/qemu-user-static>`_.
+
+Create the following ``Dockerfile``:
+
+.. code-block:: Dockerfile
+
+   FROM rust:latest 
+   RUN apt-get update && apt-get install --assume-yes --no-install-recommends \
+       libatk1.0-dev:arm64 libglib2.0-dev:arm64 libcairo-5c-dev:arm64 \
+       libpango1.0-dev:arm64  libgtk-3-dev:arm64 libsoup-3.0-dev:arm64 \
+       libjavascriptcoregtk-4.1-dev:arm64 libudev-dev:arm64 \
+       libwebkit2gtk-4.1-dev:arm64
+   RUN apt-get install --assume-yes --no-install-recommends \
+       libclang-dev:arm64 cmake:arm64 libyara-dev:arm64 libseccomp-dev:arm64
+   RUN rustup toolchain install --force-non-host nightly-aarch64-unknown-linux-gnu
+   WORKDIR /app 
+   CMD ["cargo",  "+nightly","build", "--target", "aarch64-unknown-linux-gnu", \
+        "--workspace", "--exclude", "keysas-admin", "--release" ]
+
+Allow docker to build via qemu:
+
+.. code-block:: shell-session
+   
+   $ docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+Build and run the docker image:
+
+.. code-block:: shell-session
+
+   $ docker build --platform=linux/arm64/v8 . -t keysas-build/aarch64 
+   $ docker run --platform=linux/arm64/v8 --rm -it -v "$PWD:/app" keysas-build/aarch64 
+
+The resulting files will be in the ``target/release`` directory.
+
+Until the rust image is migrated to debian 13,  you will see some errors when
+compiling the ``yara`` crate, You need to downgrade it to ``0.29.0`` (see
+`related issue
+comment <https://github.com/keysas-fr/keysas/issues/80#issuecomment-2889949214>`_).
+
+
+
 Building **Keysas-frontend**
 -----------------------------
 

